### ISSUE TITLE
Follow-up to #1437 - commands in modules and for multiplayer clients

### ISSUE THREAD

**Issue by Cervator** [MovingBlocks/Terasology#1468]
After #1437 commands in modules are seemingly not getting registered. There aren't that many of them anyway, none critical, so merged it while making this issue as a follow-up :-)

Modules affected:
* MusicDirector - MusicDirectorImpl
* NameGenerator - NameGeneratorCommands
* Hunger - HungerAuthoritySystem
* OreGeneration - OreGenRegistrySystem

I'm not sure if I missed something in fixing the compile issues, I just changed the annotation names to the new style. I figure it is an issue in the engine somewhere.

A super simple command to test would be the one in MusicDirector: "showPlaylist"

Also, @Limeth, would it be an idea to move the command registration logging from INFO to FINE or even DEBUG? Now that it works we do not need it in the startup log every run :-)

The stuff like:

```
22:03:18.820 [main] INFO  o.t.logic.console.ConsoleImpl - Command 'setSpeedMultiplier' successfully registered for class 'org.terasology.logic.debug.MovementDebugCommands'.
22:03:18.820 [main] INFO  o.t.logic.console.ConsoleImpl - Command 'sleigh' successfully registered for class 'org.terasology.logic.debug.MovementDebugCommands'.
22:03:18.820 [main] INFO  o.t.logic.console.ConsoleImpl - Command 'flight' successfully registered for class 'org.terasology.logic.debug.MovementDebugCommands'.
```

While in there keep a look out for tab characters, just noticed there are a bunch in CoreCommands - should be replaced with 4 spaces each

Also noticed a problem using commands as a client connected to a server. On executing any command you kill the server, disconnecting the client fairly gracefully (stays running, exits to main menu)

Exact error:

```
22:11:53.545 [main] ERROR o.t.p.t.TypeSerializationLibrary - Unable to register field of type Name: not a supported type or MappedContainer
22:11:53.545 [main] INFO  o.t.p.t.TypeSerializationLibrary - Unsupported field: 'engine:CommandEvent.commandName'
22:11:53.689 [main] INFO  o.t.n.internal.NetworkSystemImpl - Network shutdown
```

Works fine as host.

I made a new stable build first just in case of small issues like this (I figure probably fairly easy to fix), so merged to develop anyway to get it done already :-)

Thanks again for all the work!

**Comment by immortius** [comment]
It should be logged at the point java tries to load the annotation class for the module. I didn't check to see if this was the case.

**Comment by Cervator** [comment]
Closing as mostly fixed by #1475 with remaining multiplayer issues noted in #1482
